xbmc/network/cddb.cpp

```diff
@@ -219,8 +219,8 @@ bool Xcddb::queryCDinfo(CCdInfo* pInfo, int inexact_list_select)
 
   //##########################################################
   // Read the data from cddb
-  Recv(false); //erstmal den M�ll abholen
```

Comment:
mhh dropping the comment without replacing it with something english? By intention?